yt_dlp/extractor/abematv.py

```diff
@@ -137,13 +65,19 @@ def _get_videokey_from_ticket(self, ticket):
 
         return intlist_to_bytes(aes_ecb_decrypt(encvideokey, enckey))
 
-    def abematv_license_open(self, url):
-        url = request_to_url(url)
-        ticket = compat_urllib_parse_urlparse(url).netloc
+    @classmethod
+    def can_handle(cls, request: Request, **req_kwargs):
+        return cls._is_supported_scheme(request)
+
+    def handle(self, request: Request, **req_kwargs):
+        url = request_to_url(request)
```

Comment:
don't think we need this. request  is (at least supposed to be) always guaranteed to be a Request.

(I should probably make sure that is the case)

Reply:
Removed `request_to_url` itself instead since it is only one usage that needs it.